CHANGELOG.md

```diff
@@ -16,7 +16,62 @@ It passes a large part of the [official test suite (spectest 1.0)]:
 
 Secondly, two major optimisations are included: branch resolution is done once in the parser and not during execution; and an optimised stack implementation is introduced.
 
-In many cases this results in v0.2.0 executing code twice as fast compared to v0.1.0.
+#### Performance comparison to v0.1.0
+
+The time of WebAssembly module instantiation (including binary loading and verification) has increased by 15–30%, while the execution is approximately twice as fast.
+The execution is approximately twice as fast.
```

Comment:
This line is duplicate, because the change suggestions didnt include the two lines.